files/database/modelingScript.sql

```diff
@@ -0,0 +1,100 @@
+CREATE DATABASE LOGOS;
+-- DROP DATABASE LOGOS;
+USE LOGOS;
+
+CREATE TABLE `Category`(
+`id` BIGINT NOT NULL PRIMARY KEY AUTO_INCREMENT,
```

Comment:
Não é necessário colocar NOT NULL na coluna que foi declarada com PRIMARY KEY ela já é NOT NULL e além disso UNIQUE
https://www.mysqltutorial.org/mysql-primary-key/

Reply:
Vale para todas as tabelas!